linker/jvm/src/main/scala/org/scalajs/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -201,20 +225,28 @@ final class ClosureLinkerBackend(config: LinkerBackendImpl.Config)
 private object ClosureLinkerBackend {
   /** Minimal set of externs to compile Scala.js-emitted code with Closure. */
   private val ScalaJSExterns = """
-    /** @constructor */
```

Comment:
If these comments are not necessary anymore, we should maybe also remove them from the tree builder?

Reply:
They are still necessary for `function` declarations, which is what we use in the actual code. For externs I replaced them by `var`s so that they do not conflict with externs generated for global variables and top-level exports.